src/rest-server/deploy/rest-server.yaml.template

```diff
@@ -144,7 +146,7 @@ spec:
 {% endif %}
         ports:
         - name: rest-server
-          containerPort: 8080
+          containerPort: 9186
```

Comment:
why specify `ports` if you use `hostNetwork`

Reply:
Will remove this